package.json

```diff
@@ -33,8 +33,15 @@
     "extends": [
       "react-app",
       "react-app/jest"
+    ],
+    "ignorePatterns": [
+      "node_modules/"
```

Comment:
Changes to package.json were to silence eslint typescript errors (it was trying to lint packages in node_modules and throwing errors).  This just ensures eslint doesn't try to lint modules that we've loaded in externally.

Reply:
Got it, thanks